paasta_tools/cli/cmds/spark_run.py

```diff
@@ -536,8 +542,37 @@ def get_spark_app_name(original_docker_cmd: Union[Any, str, List[str]]) -> str:
 
     if spark_app_name is None:
         spark_app_name = "paasta_spark_run"
-
     spark_app_name += f"_{get_username()}"
+    if autogen != '-1':
+        document = f"""
```

Comment:
I would move this at the top of the module, as a constant. It seems also that you're programmatically changing some values inside the string, I would instead use `.format()` function